unified-scheduler-pool/src/lib.rs

```diff
@@ -782,7 +781,7 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
             session_result_receiver,
             session_result_with_timings: None,
             scheduler_thread: None,
-            handler_threads: Vec::with_capacity(handler_count),
+            handler_threads: vec![],
```

Comment:
`handler_threads` is reassigned in `start_threads`, so no need to reserve capacity. also, this isn't perf sensitive code.